ui/src/i18n/en.json

```diff
@@ -363,7 +363,14 @@
                 "defaultView": "Default View",
                 "desktop_notifications": "Desktop Notifications",
                 "lastfmScrobbling": "Scrobble to Last.fm",
-                "listenBrainzScrobbling": "Scrobble to ListenBrainz"
+                "listenBrainzScrobbling": "Scrobble to ListenBrainz",
+                "replaygain": "ReplayGain Mode",
+                "preAmp": "ReplayGain PreAmp (dB)",
+                "gain": {
+                    "none": "Disabled",
+                    "album": "Normalize albums",
+                    "track": "Normalize tracks"
+                }
```

Comment:
Suggestions for better descriptions here are appreciated

Reply:
I'd prefer "Use Album Gain" and "Use Track Gain". 
Thats f.e. how the DSub Android Client states it

Reply:
Yep, makes sense @metalheim